File: 167_Two_Sum_II.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int left = 0 , right = nums.length-1;
        while(left < right){
            int sum = nums[left] + nums[right];
            if(sum == target){
                return new int[]{left +1 , right+1};
            }
            else if(sum > target){
                right--;
            }  
            else if(sum < target){
                left++;
            }
        }
        return new int[]{-1,-1};
    }
}

File: 167_TwoSum_II.js
function twoSum(numbers,target){
    let left = 0 
    let right = numbers.length-1
    while(left < right){
        const twoSum = numbers[left] + numbers[right]
        if(twoSum > target){
            right--
        }else if (twoSum < target){
            left--
        }else{
            return [left+1 , right+1]
        }
    }
}
numbers = [2,3,4], target = 6
console.log(twoSum(numbers,target));